### DISCUSSION TITLE
Reuse existing Prism Container Registrations in an Microsoft.AspNetCore WebService

### DISCUSSION THREAD

**Discussion by dockerandy** [PrismLibrary/Prism#2892] [General]
Dear Prism Team,
we have an existing Prism WPF Application with a lot of Service Registrations in Prim IContainerRegistry [override Methode protected override void RegisterTypes(IContainerRegistry containerRegistry)]. 

Is it possible to reuse this Prism Container Registrations in an Microsoft.AspNetCore WebService for Dependency Injection?

I have only found that you can reuse existing registrations in IServiceCollections in an PrismContainer (Methode Prism.Container.Extensions.RegisterServices().

or use a manual Resolving like this Way:
- i've registered a containerExtension in IServiceCollection
- Register all necessary Interfaces in the IContainerExtension 
- The Controller gets the IContainerExtension injected
- Service Resolving inside the Constructor of the Controller like this:
IMyService myService = containerExtension.Resolve\<IMyService\>();

Best Regards
Andy

**Comment by dansiegel** [comment]
It'd be pretty easy to implement IContainerRegistry around IServiceCollection.
